e2e/go.mod

```diff
@@ -1,17 +1,18 @@
 module github.com/cosmos/ibc-go/e2e
 
-go 1.18
+go 1.19
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 require (
-	github.com/cosmos/cosmos-sdk v0.46.2
+	github.com/cosmos/cosmos-sdk v0.46.1
+	github.com/cosmos/gogoproto v1.4.2
```

Comment:
We are blocked on ICS23 to upgrading to cosmos/gogoproto for fixing the last tests.

Reply:
For my understanding. Is the issue because of the proto registration happens at [`init`](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/types/channel.pb.go#L489) and now that the imports are changing, the ics23 Spec is not being registered the same [global registry](https://github.com/gogo/protobuf/blob/v1.3.2/proto/properties.go#L535)? A chain is essentially expected to use a single proto go module for all its types

Reply:
It looks like cosmos/ics23 still needs this change. Will you handle it or should we?

Reply:
Now looking at the repo it's only a question of tagging a 0.10.